transactions/src/main/resources/schema/wildfly-txn_3_1.xsd

```diff
@@ -171,7 +171,7 @@
             </xs:documentation>
         </xs:annotation>
         <xs:attribute name="path" type="xs:string" default="tx-object-store"/>
-        <xs:attribute name="relative-to" type="xs:string" default="jboss.server.data.dir" />
+        <xs:attribute name="relative-to" type="xs:string"/>
```

Comment:
No, as the 3.1 xsd is new for WildFly 11, added in https://github.com/wildfly/wildfly/pull/8902